program/src/main/System.vala

```diff
@@ -1,12 +1,14 @@
 //static class that has main method and some others
-public class System
-{
+public class System {
+
     public static unowned Gtk.Window? mainWindow;
     public static Gtk.EventBox examEventBox;
     public static ExamImage examImage;
     public static Gee.ArrayList<QuestionSet> examQuestionSet; //the master list of all QuestionSets for the current exam
     public static int examPagesPerTest = 0;
     public static int examQuestionsPerTest = 0;
+    public static int question_incrementer = 0; //For the purpose of testing new set up process. Will remove and replace with examQuestionsPerTest
```

Comment:
Should this be replaced at this point then?

Reply:
Looking over it, I think that question_incrementer actually ended up needing to be its own thing because there were other parts of the code that were referencing examQuestionsPerTest that I didn't want to mess with. I might look into this more in the future but I think having a separate variable at least for the clarity and separation of concerns should be permissible in this case